packages/javascript/bh-shared-ui/src/components/FinishedJobsTable/finishedJobs.test.ts

```
@@ -0,0 +1,180 @@
+        status: (index % 10) - 1,
+        session_collection: randBoolean(),
+        local_group_collection: randBoolean(),
+        ad_structure_collection: randBoolean(),
+        cert_services_collection: randBoolean(),
+        ca_registry_collection: randBoolean(),
+        dc_registry_collection: randBoolean(),
+        all_trusted_domains: randBoolean(),
+    })),
+    limit: 10,
+    skip: 10,
+};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)